pkg/assets/create/creater.go

```diff
@@ -57,45 +55,50 @@ func EnsureManifestsCreated(ctx context.Context, manifestDir string, restConfig
 		options.StdErr = os.Stderr
 	}
 
-	// Default QPS in client (when not specified) is 5 requests/per second
-	// This specifies the interval between "create-all-resources", no need to make this configurable.
-	interval := 200 * time.Millisecond
-
-	// Retry creation until no errors are returned or the timeout is hit.
 	var (
 		lastCreateError      error
-		retryCount           int
 		mapper               meta.RESTMapper
 		needDiscoveryRefresh bool = true
 	)
-	err = wait.PollImmediateUntil(interval, func() (bool, error) {
```

Comment:
`PollImmediate` is project standard.

Reply:
> `PollImmediate` is project standard

Even when [you don't want to sleep between iterations][1]?

[1]: https://github.com/openshift/library-go/pull/226#discussion_r254237203

Reply:
I can make that sleep 10ms ;-) But i wanted it to match the default client QpS (5 which means 200ms between requests). So a retry will wait the same time as client will do. Again, I don't have problem changing that interval to 10ms (I doubt we ever will have problems when we DoS API server with create calls).

Reply:
> I can make that sleep 10ms ;-)

It's still a wasted ticker goroutine ;).

> But i wanted it to match the default client QpS (5 which means 200ms between requests). So a retry will wait the same time as client will do.

But if we'd made a client request, we'd already get the client's internal limiting, right?  If you dump logs for this PR's tests as they stand, you see I'm not flooding fake client requests even without an external sleep.

Reply:
In real life, that interval as it is now, will likely never cause any problem or extensive delay (unless we plan to use this helper to create >500 manifests). In real life, pulling updated discovery from the server will probably take seconds already and every non-CR resource will be created at the first run. So whether we remove this interval or set it to 10ms or 400ms won't make that much difference ;-)

Reply:
> So instead of writing custom polling and cancellation logic, I would prefer to follow the k8s style.

A `for` loop is not custom polling logic ;).  The `select` for cancelation checking is also a standard approach, and seems simpler to me to inline when you don't need a wait vs. calling `PollImmediateUntil` just to wrap this one `Context` `select`.  But I'm not a maintainer ;).  You're all sure you prefer `PollImmediateUntil` over `for`?

Reply:
yeah, PollImmediateUntil is k8s standard as @deads2k already mentioned.